react/MonthPicker/NativeMonthPicker/NativeMonthPicker.js

```diff
@@ -81,18 +86,22 @@ export default class NativeMonthPicker extends Component {
 
     return (
       <div className={rootClasses}>
+        <FieldLabel
+          {...{ id: `${id}`, label: label ? <span>{label} <ScreenReaderOnly>Month Year</ScreenReaderOnly></span> : <ScreenReaderOnly>Month Year</ScreenReaderOnly>, labelProps, secondaryLabel, tertiaryLabel }}
```

Comment:
No need to use a template string for `id` here. And as above, it'd be great to format this a little friendlier